eve/main.yml

```diff
@@ -0,0 +1,48 @@
+---
+version: 0.2
+
+branches:
+  default:
+    stage: pre-merge
+
+models:
```

Comment:
you don't need models since it is used only once, it would be clearer with a flat yaml

Reply:
I've removed the model